fcc-test-suite-template-master/css/style.css

```diff
@@ -0,0 +1,70 @@
+* {
+  margin: auto;
+  padding: auto;
+  text-align: center;
+}
+#video {
+  margin-top: 100px;
+  width: 500px;
+}
+#header {
+  margin-top: 30px;
+  height: 100px;
+}
+#header-img {
+  width: 230px;
+  height: 90px;
+  margin-left: 190px;
+}
+input {
+  width: 260px;
+  height: 30px;
+  border-radius: 2em;
+}
+input:hover {
+  background-color: beige;
+}
+h1 {
+  margin-bottom: 20px;
+  margin-top: 70px;
+}
+
+#submit {
+  width: 80px;
+  height: 31px;
+  border-radius: 1em;
+}
+#submit a {
+  text-decoration: none;
+}
+header {
+  background-color: antiquewhite;
+}
+.nav-link {
+  display: flex;
+  float: left;
+  margin-left: 30px;
+  width: 30px;
+  height: 40px;
+  color: black;
+  text-decoration: none;
+}
+.nav-link:hover {
+  height: 100px;
+  color: green;
+}
+
```

Comment:
@SAZZAD1Q2   It's all right you did 👍 . You have another simple requirement. That you may use **linters** to check 🔋 whether your is error-free or not.